src/Nox.Core/Infrastructure/Messaging/AzureServiceBus/AzureServiceBusBrokerProvider.cs

```diff
@@ -32,7 +33,7 @@ public AzureServiceBusBrokerProvider(NoxSolution noxSolution)
 
             cfg.ConfigureEndpoints(context);
             cfg.UseRawJsonSerializer();
-            cfg.MessageTopology.SetEntityNameFormatter(new CustomEntityNameFormatter(_noxSolution.PlatformId, _noxSolution.Name));
+            cfg.MessageTopology.SetEntityNameFormatter(new CustomEntityNameFormatter(_noxSolution.PlatformId, _noxSolution.Name, environmentName.ToLower()));
```

Comment:
SetEntityNameFormatter is already normalizing the environment name, no need to leak this sort of normalization to outside
```suggestion
            cfg.MessageTopology.SetEntityNameFormatter(new CustomEntityNameFormatter(_noxSolution.PlatformId, _noxSolution.Name, environmentName));
```